velox/dwio/dwrf/common/CachedBufferedInput.cpp

```diff
@@ -15,14 +15,31 @@
  */
 
 #include "velox/dwio/dwrf/common/CachedBufferedInput.h"
+#include "velox/common/process/TraceContext.h"
 #include "velox/dwio/dwrf/common/CacheInputStream.h"
 
+DEFINE_int32(cache_load_quantum, 8 << 20, "Max size of single IO to cache");
+DEFINE_int32(
+    cache_prefetch_min_pct,
```

Comment:
This logic doesn't seem to be covered by PR description. Can it be moved into a separate PR?